cmd/clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -225,15 +225,15 @@ func (d *ClusterDeployer) updateClusterEndpoint(client clusterclient.Client, pro
 	// Update cluster endpoint. Needed till this logic moves into cluster controller.
 	// TODO: https://github.com/kubernetes-sigs/cluster-api/issues/158
 	// Fetch fresh objects.
-	cluster, master, _, err := getClusterAPIObject(client, clusterName, namespace)
+	cluster, controlplane, _, err := getClusterAPIObject(client, clusterName, namespace)
 	if err != nil {
 		return err
 	}
-	masterIP, err := provider.GetIP(cluster, master)
+	controlplaneIP, err := provider.GetIP(cluster, controlplane)
 	if err != nil {
-		return fmt.Errorf("unable to get master IP: %v", err)
+		return fmt.Errorf("unable to get controlplane IP: %v", err)
```

Comment:
"unable to get the control plane IP" or "unable to get the IP of the control plane" or "unable to get the IP of the cluster endpoint"